tools/wpt/expectation.json

```diff
@@ -3241,13 +3241,7 @@
     },
     "data-urls": {
       "base64.any.html": true,
-      "base64.any.worker.html": true,
-      "processing.any.html": [
-        "\"data:text/plain;a=\\\",\\\",X\""
-      ],
-      "processing.any.worker.html": [
-        "\"data:text/plain;a=\\\",\\\",X\""
-      ]
```

Comment:
It seems like these should be removed? That resolved the CI failure. I'm guessing this is a fix in data-url? cc @lucacasonato

```rs
use data_url::DataUrl;

let url = DataUrl::process("data:text/plain;a=\",\",X").unwrap();
let (body, fragment) = url.decode_to_vec().unwrap();
eprintln!("Type: {:?}", url.mime_type());
eprintln!("Body: {:?}", String::from_utf8(body).unwrap());
```

Output in 0.1.1:
```
Type: text/plain - Mime { type_: "text", subtype: "plain", parameters: [] }
Body: [34, 44, 88]
```

Then in 0.2.0:

```
Type: text/plain;a="" - Mime { type_: "text", subtype: "plain", parameters: [("a", "")] }
Body: [34, 44, 88]
```